app/models/container_project.rb

```diff
@@ -36,5 +36,18 @@ def containers_count
     container_definitions.size
   end
 
+  include EventMixin
+
   acts_as_miq_taggable
+
+  def event_where_clause(assoc = :ems_events)
+    case assoc.to_sym
+      when :ems_events
+        # TODO: improve relationship using the id
+        ["container_namespace = ? AND ems_id = ?", name, ems_id]
```

Comment:
@dkorn you were still seeing the `ems_id` ambiguous error because you forgot `#{events_table_name(assoc)}`:

    ["container_namespace = ? AND #{events_table_name(assoc)}.ems_id = ?", name, ems_id]